### PR TITLE
Fixed README.md and readme.txt; new screenshots

### PR DESCRIPTION
**Pull request by joshcanhelp**
README.md (used here on GH):

- Removed duplicate and out-of-date documentation in favor of what's on auth0.com/docs
- Removed screenshots
- Removed 3.5 note
- Added more about contributing, including install instructions

readme.txt (used on WP.org plugin page):

- Removed duplicate/out-of-date installation instructions, point to auth0.com/docs
- Updated screenshots
- Added an FAQ about support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,51 +5,50 @@ Login by Auth0
...
+* Make sure to test against the lowest PHP version supported (see `Requires PHP:` [here](https://github.com/auth0/wp-auth0/blob/master/readme.txt#L5)) 
 
-You can use this feature with API's provided by plugins like **WP REST API (WP API)**.
+### How to install and configure WordPress for testing
 
-## Contributing
+We try to cover as many use cases as possible and one way to do that is with a "headless" WordPress install (core WordPress files in a separate directory). Here's a quick and easy way to set that up on the command line:
 
-Thank you in advance!
+1. `mkdir wp-doc-root; cd wp-doc-root; mkdir wp;`
```
Comment (lbalmaceda) [inline README.md]: might want to put them all inside a code block to highlight the syntax

````bash
mkdir wp-doc-root; cd wp-doc-root; mkdir wp;
mkdir wp-content; mkdir wp-content/themes; mkdir wp-content/plugins;
git clone https://github.com/WordPress/WordPress.git wp; # takes a while
cp ./wp/wp-config-sample.php ./wp-config.php; # edit to add MySQL DB info and set "WP_DEBUG" to TRUE

# You can even have one comment per line to be more clear about what the command is doing

cp ./wp/index.php ./index.php; # replace "/wp-blog-header.php" with "/wp/wp-blog-header.php"
git clone https://github.com/joshcanhelp/auth0-wp-test.git ./wp-content/themes/auth0-wp-test
````

File: README.md [hunk 2]
```diff
@@ -65,27 +64,13 @@ To watch and auto-compile it while working:
...
-![](https://raw.githubusercontent.com/auth0/wp-auth0/master/screenshot-4.png)
-
-![](https://raw.githubusercontent.com/auth0/wp-auth0/master/screenshot-5.png)
-
-![](https://raw.githubusercontent.com/auth0/wp-auth0/master/screenshot-6.png)
-
 ## Issue Reporting
 
-If you have found a bug or if you have a feature request, please report them at this repository issues section. Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
+If you find a bug or if you have a feature request, please report them in [Issues for this repo](https://github.com/auth0/wp-auth0/issues). Please do not report security vulnerabilities in a public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
```
Comment (lbalmaceda) [inline README.md]: I think it looks better this way:

...report them in the [issues tab](https://github.com/auth0/wp-auth0/issues)

File: readme.txt [hunk 3]
```diff
@@ -31,22 +31,16 @@ This plugin gives WordPress a new Login Widget (powered by [Auth0](https://auth0
...
-
-1. Install from the WordPress Store or upload the entire `wp-auth0` folder to the `/wp-content/plugins/` directory.
-1. Activate the plugin through the 'Plugins' menu in WordPress.
-1. Create an account in Auth0 (https://auth0.com) and add a new PHP Application. Copy the Client ID, Client Secret and Domain from the Settings of the Application.
-1. On the Settings of the Auth0 application change the Callback URL to be: `http://your-domain/index.php?auth0=1`. Using **TLS/SSL** is **recommended for production**.
-1. Go back to WordPress `Settings` - `Auth0 Settings` edit the *Domain*, *Client ID* and *Client Secret* with the ones you copied from Auth0 Dashboard.
+Please see the Auth0 Docs site for [complete installation and configuration instructions](https://auth0
+.com/docs/cms/wordpress/installation).
 
 == Screenshots ==
```
Comment (lbalmaceda) [inline readme.txt]: What is this section for, if you have removed the embedded ones above?

Comment (joshcanhelp) [reply readme.txt]: Standard section folks look for on the WP.org repo

File: README.md (hunk 1 above)
Comment (joshcanhelp) [reply README.md]: Just had this as a list for order-of-ops but I like the block better

File: README.md [hunk 4]
```diff
@@ -5,51 +5,50 @@ Login by Auth0
...
+## Extending the plugin
 
-After the user logs in via Auth0 in your Api client (ie: using Lock in a mobile app) you will get a JWT (JSON Web Token). Then you need to send this token in a HTTP header in the following way:
+Please see the [Extending page on auth0.com/docs](https://auth0.com/docs/cms/wordpress/extending) for documentation on existing hooks.
 
-```
-Authorization: Bearer ##jwt##
-```
+We're happy to review and approve new filters and actions that help you integrate even further in this plugin. Please
+ see the **Contributing** section below for more information.
```
[the remark this answers is not in the document]
Comment (lbalmaceda) [reply README.md]: Probably unwanted _new line_ too.

File: README.md [hunk 5]
```diff
@@ -66,7 +73,7 @@ $ stylus -w -o initial-setup.css initial-setup/main.styl
 
 ## Issue Reporting
 
-If you find a bug or if you have a feature request, please report them in [Issues for this repo](https://github.com/auth0/wp-auth0/issues). Please do not report security vulnerabilities in a public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
+If you find a bug or if you have a feature request, please report them in the [Issues tab](issues). Please do not report security vulnerabilities in a public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
```
Comment (lbalmaceda) [inline README.md]: I don't think this link is going to work. Better put the full URL here.

File: README.md [hunk 6]
```diff
@@ -34,18 +31,28 @@ We're happy to review and approve new filters and actions that help you integrat
 with a feature add or bug fix are difficult to review and may be rejected
 * Use the latest version of WordPress and turn `WP_DEBUG` on
 * If other plugins are installed during testing that might affect behavior, please list those in the PR description
-* Make sure to test against the lowest PHP version supported (see `Requires PHP:` [here](https://github.com/auth0/wp-auth0/blob/master/readme.txt#L5)) 
+* Make sure to test against the lowest PHP version supported (see `Requires PHP:` [here](blob/master/readme.txt#L5))
```
Comment (lbalmaceda) [inline README.md]: This won't work. Either full URL or relative from the root like this -> `readme.txt`. Note using "relative" won't work for different branches than the default one.

**Pull request merged**